src/java/org/apache/ivy/plugins/repository/sftp/SFTPRepository.java

```diff
@@ -148,7 +142,7 @@ public void put(File source, String destination, boolean overwrite) throws IOExc
         }
     }
 
-    private void mkdirs(String directory, ChannelSftp c) throws IOException, SftpException {
+    private void mkdirs(String directory, ChannelSftp c) throws SftpException {
```

Comment:
Similar comment as here https://github.com/apache/ant-ivy/pull/52/files#r127395633 applies to this line